src/plugins/serverless/.i18nrc.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
nit: this file is not needed for built-in plugins... I'll create an issue to make sure we update the plugin generator accordingly